var/spack/repos/builtin/packages/vesta/package.py

```diff
@@ -0,0 +1,30 @@
+from spack import *
+import os
+import fnmatch
+import shutil
+
+
+class Vesta(Package):
+    """VESTA is a 3D visualization program for structural models, volumetric data
+       such as electron/nuclear densities, and crystal morphologies."""
+
+    homepage = "http://jp-minerals.org/vesta"
+    url      = "https://jp-minerals.org/vesta/archives/3.4.6/VESTA-x86_64.tar.bz2"
+
+    version('3.4.6', '1d4651e86193f305831aa5db3dcfe789')
+
+    depends_on('gtkplus@2.1.0:')
+    depends_on('mesa')
+    depends_on('cairo@1.0:')
+    depends_on('gcc@5.4.0:')
```

Comment:
Compilers are not dependencies in a formal sense. I think what you want is:
```python
conflicts('%gcc@:5.3')
```

Reply:
done